Resources/Prototypes/Entities/Clothing/Masks/masks.yml

```diff
@@ -397,8 +397,8 @@
   - type: Armor
     modifiers:
       coefficients:
-        Blunt: 0.90
-        Slash: 0.90
+        Blunt: 0.95 # Frontier 0.90<0.95
+        Slash: 0.95 # Frontier 0.90<0.95
```

Comment:
Should the swat gas mask also be brought into line with this?

Reply:
sure